ci/test/pipeline.template.yml

```diff
@@ -39,18 +39,18 @@ steps:
       queue: builder-linux-aarch64
     coverage: skip
 
-  - id: check-merge-with-target
-    label: Merge skew cargo check
-    command: ci/test/check-merge-with-target.sh
-    inputs:
-      - Cargo.lock
-      - "**/Cargo.toml"
-      - "**/*.rs"
-    timeout_in_minutes: 30
-    agents:
-      queue: linux-x86_64
-    if: "build.pull_request.id != null"
-    coverage: skip
+  #- id: check-merge-with-target
```

Comment:
This is all just for quicker turnaround in my testing, will uncomment if we want this approach.